unic/ucd/name/src/name.rs

```diff
@@ -78,21 +106,49 @@ impl Name {
     fn number_of_hex_digits(ch: char) -> usize {
         (32 - u32::leading_zeros(ch as u32) as usize + 3) / 4
     }
+
+    fn jamo_short_name(ch: char) -> &'static str {
+        data::JAMO_SHORT_NAMES
+            .find(ch)
+            .expect("Unexpected jamo character")
+    }
+
+    fn collect_jamo_short_names(ch: char) -> [Option<&'static str>; 3] {
+        let mut jamos = [None; 3];
+        {
+            let mut index = 0;
+            let mut collect_jamos = |jamo| {
+                jamos[index] = Some(Name::jamo_short_name(jamo));
```

Comment:
Can we add a (`debug_`?)`assert!` here for the bound on `index`? If max number of short names turns out to be higher it'd be better to have an assertion fail and say what overflowed rather than just an out-of-index, even if it's pointing here.

Reply:
Added a `debug_assert!`, thanks!